spec/support/corpus.rb

```diff
@@ -134,15 +138,33 @@ def install_mutant
         system(%w[bundle])
       end
 
-      # Not in the docs. Number from chatting with their support.
-      CIRCLE_CI_CONTAINER_PROCESSES = 2
+      # The effective ruby file paths
+      #
+      # @return [Array<Pathname>]
+      def effective_ruby_paths
+        paths = Pathname
+          .glob(repo_path.join(RUBY_FILE_PATTERN))
+          .sort_by(&:size)
+          .reverse
+
+        paths - excluded_paths
+      end
+
+      # The excluded file paths
+      #
+      # @return [Array<Pathname>]
+      def excluded_paths
+        exclude.flat_map do |pattern|
+          Pathname.glob(repo_path.join(pattern))
```

Comment:
Initially I wanted to avoid this, as there is no `PathGlob.union` thing, but now as I see it I like it more. As interpolating via `#{}` next to literal `{}` is uneasy to read I think we should go for: `'{%s}' % exclude.join(',')` as this avoids the confusion when parsing the code by brain. Than move the format into a constant.